requirements-dev.txt

```diff
@@ -1,6 +1,6 @@
 -r requirements.txt
-addict
 black
 flake8
+lintrunner
+lintrunner-adapters
```

Comment:
do we need remove line https://github.com/microsoft/Olive/blob/main/.github/workflows/lint.yml#L46?

Reply:
Removed.